SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -394,6 +402,14 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 				return;
 			}
 
+			// If the received actor is torn off, don't bother receiving it.
+			if (EntityActor->GetTearOff())
```

Comment:
stretch goal - can we check tearoff in the component data, and save the unnecessary create/destroy?

Reply:
Can look into it 👍